src/modules/PublicLab.TagsModule.js

```diff
@@ -89,6 +89,17 @@ module.exports = PublicLab.TagsModule = PublicLab.Module.extend({
       _module.focusables.push(_module.el.find('.tokenfield .tt-input'));
 
 
+      if(window.innerWidth <= 992) {
+        _module.el.find('.tokenfield').addClass('input-area');
```

Comment:
The 2 "ifs" are basically the same, except this one adds input-area class on `.tokenfield`.
Can't this be in src/PublicLab.Editor.js?

Reply:
This code never worked in src/PublicLab.Editor.js in all the ways I tweaked it, that's why I brought it here. 

I should try some other way perhaps and see if it works. And also work on the other changes you suggested.

Thank you.